opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -113,6 +138,17 @@
          */
         SpanBuilder addReference(String referenceType, SpanContext referencedContext);
 
+        /**
+         * Remove any explicit (e.g., via {@link SpanBuilder#addReference(String,SpanContext)}) or implicit (e.g., via
+         * {@link ActiveSpanSource#active}) references to parent / predecessor SpanContexts, thus making the built
+         * Span a "root" of a Trace tree/graph.
+         *
+         * <p>
+         * Subsequent calls to {@link SpanBuilder#addReference(String, SpanContext)} /
+         * {@link SpanBuilder#asChildOf(Span)} / etc are permitted and behave as per usual.
+         */
+        SpanBuilder asRoot();
```

Comment:
The documentation implies this actually being something like `withoutReferences()`.
However `asRoot()` communicates a more explicit intent. I think in that case the concept of `root` shouldn't be within double quotes in the documentation (implying not _really_ root, but something similar).

*) Sorry for using so many words (my non-native English hurts me here).
I guess I'm saying either embrace the _root_ concept or choose a name that fits.